src/libsync/owncloudpropagator.cpp

```diff
@@ -369,15 +370,36 @@ qint64 OwncloudPropagator::smallFileSize()
     return smallFileSize;
 }
 
-void OwncloudPropagator::start(const SyncFileItemVector &items)
+void OwncloudPropagator::start(const SyncFileItemVector &_items)
```

Comment:
is that underscore intentional? I usually use the underscore for member variables only.

Reply:
I create a copy one line later and well...